Source/Core/VideoCommon/AbstractGfx.cpp

```diff
@@ -87,6 +98,16 @@ void AbstractGfx::ClearRegion(const MathUtil::Rectangle<int>& target_rc, bool co
   EndUtilityDrawing();
 }
 
+void AbstractGfx::SetViewportAndScissor(ViewportAndScissors viewport_and_scissors)
+{
+  m_viewport_and_scissors = std::move(viewport_and_scissors);
```

Comment:
`ViewportAndScissors` is trivially-copyable so `std::move` has no effect here. It produces a clang-tidy warning.